sdk/resourcemanager/Azure.ResourceManager.Core/tests/SessionRecords/ResourceGroupContainerTests/Get().json

```diff
@@ -16,15 +17,15 @@
         "Cache-Control": "no-cache",
         "Content-Length": "450",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Fri, 11 Jun 2021 22:58:29 GMT",
```

Comment:
this file appears to have only time / id based changes can we revert it? along with StartExportTemplate, Update, and StartDelete